src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapper.java

```diff
@@ -0,0 +1,47 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.InstantType;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.UriType;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.PathologyRecord;
+import uk.nhs.digital.nhsconnect.lab.results.utils.UUIDGenerator;
+
+import java.util.List;
+
+@Component
+public class BundleMapper {
+    private static final List<UriType> BUNDLE_META_PROFILE = List.of(new UriType("https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1"));
+    private static final String BUNDLE_IDENTIFIER_SYSTEM = "https://tools.ietf.org/html/rfc4122";
+    private static final String FULL_URL_PREFIX = "urn:uuid:";
+
+    public Bundle mapToBundle(PathologyRecord pathologyRecord) {
+        Bundle bundle = generateInitialPathologyBundle();
```

Comment:
Interesting. Feel like this is something else we should agree as a team and add to our coding standards, as we're probably very inconsistent with this

Reply:
NHAIS project uses them _looooooots_; imo too much. But again, there's nothing documented one way or the other.

I maintain there are a couple of disjunctive criteria for using `var`:
* the type is immediately obvious (as in this case), or
* the type doesn't matter (because it's "passed through" without really being inspected at all, perhaps).

An arguable third would be when the semantic purpose of the variable - as indicated by its name and context - is more important than its type, and the actual uses of the variable are clear.

Reply:
Now, this on the other hand, is not something i'm taking advantage of myself. I like to declare explicit types. However, for this particular case, I can see the attraction - the type is obvious. But, it's personal choice.

Reply:
Yeah this example it's very clear what the type is, but I prefer being explicit all the time, rather than mixing between `var` and the actual type, it doesn't hurt. But that's my preference. What do yous all think? @saileshp-kainos @antonlebedjko

Reply:
I prefer declaring explicit types over using `var` but like @andrewwa-kainos & @n-r-b have said, in this case it's clear what the type is.

Reply:
I would go for the explicit types as it's just much more clear for me. I know that in this example it's also clear what the type is but still, personally I would prefer explicit types